terraform/modules/fake_server_resources/fake_server_resources.tf

```diff
@@ -15,7 +15,7 @@ variable "sum_part_bucket_writer_email" {
 }
 
 variable "ingestors" {
-  type = map(string)
+  type = list(string)
```

Comment:
Are we sure we're not going to need the actual ingestor addresses here in the future?

Reply:
If we do, then we can change this back to a richer representation.